std_msgs/Header header
laundryman/CubeInfo[] data